File: Data Science/Assignment 1.py
import pandas as pd

# Load CSV dataset from the internet
url = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"

df = pd.read_csv(url)

# Display first 5 rows
print("First 5 Rows:")
print(df.head())

# Show column names
print("\nColumn Names:")
print(df.columns.tolist())

# Display number of rows and columns
print("\nNumber of Rows:", df.shape[0])
print("Number of Columns:", df.shape[1])

# Show summary statistics
print("\nSummary Statistics:")
print(df.describe(include='all'))

File: Data Science R Language/assignment1.R
# Assignment 1

# Load dataset from the internet
url <- "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"

df <- read.csv(url)

# Display first 5 rows
cat("First 5 Rows:\n")
head(df, 5)

# Display column names
cat("\nColumn Names:\n")
colnames(df)

# Display number of rows and columns
cat("\nNumber of Rows:", nrow(df), "\n")
cat("Number of Columns:", ncol(df), "\n")

# Display summary statistics
cat("\nSummary Statistics:\n")
summary(df)